### DISCUSSION TITLE
How to get related taxonomies data into blade tempalate

### DISCUSSION THREAD

**Discussion by rinkiqa12** [statamic/cms#4339] [Q&A]
Hello All,

I'm totally fresher in Statamic CMS. I just want to know how to get taxonomies data related to other taxonomies data selected?
My current taxonomies  .yaml file is created this:-

title: 'HPS ABC'
content: 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo'
updated_by: 1
blueprint: product_type
**commodity_tag**:
  - hemp
  - seed

This **commodity_tag** is related to other taxonomies data. And I want to show **commodity_tag** data into my blade template engine. Please help me I already search on google but have not been able to found out this solution.

Thanks,

**Comment by edalzell** [comment]
Have you seen this [addon](https://statamic.com/addons/silentz/blade)?

**Reply by rinkiqa12** [reply]
Yes, I have already added and implemented some other dynamically data on blade view. But in this case, there are related to each other taxonomies.

I am doing a dump and die to **commodity_tag** on blade view and got the array.

Statamic\Fields\Value {#1947 ▼
  #raw: array:2 [▼
    0 => "hemp"
    1 => "seed"
  ]
  #handle: "commodity_tag"
  #fieldtype: Statamic\Fieldtypes\Terms {#1998 ▼
    #statusIcons: false
    #taggable: true
    #icon: "taxonomy"
    #component: "relationship"
    #indexComponent: "relationship"
    #itemComponent: "related-item"
    #formComponent: null
    #categories: array:1 [▶]
    #canEdit: false
    #canCreate: false
    #canSearch: false
    #defaultValue: []
    #formComponentProps: array:1 [▶]
    #field: Statamic\Fields\Field {#2010 ▶}
    #localizable: true
    #validatable: true
    #defaultable: true
    #selectable: true
    #selectableInForms: false
    #rules: []
    #extraRules: []
    #configFields: []
  }
  #augmentable: Statamic\Taxonomies\LocalizedTerm {#2015 ▼
    #locale: "default"
    #term: Statamic\Taxonomies\Term {#2003 ▶}
    #slug: null
    #published: true
    #selectedQueryColumns: null
    #supplements: Illuminate\Support\Collection {#1916 ▶}
  }
  #shallow: false
}

I also tried foreach loop for **commodity_tag** on blade view. But not able to get the data. Please let me know where I am going to wrong.

Thanks

**Comment by rinkiqa12** [comment]
Anyone, please help what should I do?
